memberbooth.py

```diff
@@ -40,7 +41,13 @@ def main():
         logger.error("The makeradmin client is not logged in")
         sys.exit(-1)
 
-    app = Application(makeradmin_client)
+    try:
+        key_reader = EM4100_KeyReader.get_reader()
+    except NoReaderFound as e:
```

Comment:
If --development is used, I think that the application should run even if there is no reader attached.

Reply:
The function below also has to be changed to do nothing on timer expired, or maybe even simpler, never start the timer if the --development flag is specified. 


https://github.com/makerspace/memberbooth/pull/35/files#diff-17273fda14cf37080e22b106abbe439eR99

Reply:
I agree

Reply:
I think we should add back the regular keyboard input (that worked with the Aptus keyboard). However, it is better if this is a separate flag (like `--tag-reader-type=APTUS`, `--tag-reader-type=MANUAL` and `--tag-reader-type=EM4100`). The `--development` flag is stated to be for events.

Reply:
Could you add that as an issue and add it to the --development flag for now? I think the --tag-reader flag requires a bigger update to the states.py file.

Reply:
> If --development is used, I think that the application should run even if there is no reader attached.

But what should it do? Just mock the whole reader and not allow input? Then that could be a separate flag `--no-reader`? (Like the other mock flags)